stagecraft/apps/datasets/tests/models/test_data_set.py

```diff
@@ -69,6 +72,31 @@ def test_saving_with_new_name_doesnt_change_it(self):
         data_sets = DataSet.objects.filter(name='data_group1_data_type1')
         assert_equal(len(data_sets), 1)
 
+    def test_can_create_dataset_with_owner(self):
+        DataSet.objects.create(
+            data_group=self.data_group1,
+            data_type=self.data_type1)
+        data_set1 = DataSet.objects.get(name='data_group1_data_type1')
+        user, _ = User.objects.get_or_create(
+            email='foobar.lastname@gov.uk')
+        data_set1.save()
+        data_set1.owners.add(user)
+
+        assert_equal('foobar.lastname@gov.uk', data_set1.owners.first().email)
+
+    def test_can_create_dataset_without_owner(self):
+        DataSet.objects.create(
+            data_group=self.data_group1,
+            data_type=self.data_type1)
+        data_set1 = DataSet.objects.get(name='data_group1_data_type1')
+        data_set1.save()
+
+        assert_equal(len(data_set1.owners.all()), 0)
+
+    def test_can_create_dataset_without_owner_in_forms(self):
```

Comment:
I was really testing it was set, and testing it was true to be on the safe side.  As it not being set was what had caused the problems, and that this the only thing that has actually been changed besides the tests.  It might be too much, but I was also adding the test on the assumption that there can't be too many tests?